### PR TITLE
Add Cannery

### PR DESCRIPTION
**Pull request by shibaobun**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

**Comment by kokomo123** [comment]
Hi there, feel free to resubmit as this doesn't seem right, feel free to use the templates listed here: https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/CONTRIBUTING.md

**Comment by shibaobun** [comment]
@kokomo123 this is a bit of a weird request, but could you reopen this PR for a moment? Thank you!

**Comment by kokomo123** [comment]
@shibaobun unfortunately, this can't be reopened since u already made a new one. Apologies

**Comment by shibaobun** [comment]
@kokomo123 this is a special case, and I want to close it immediately afterwards, would that be okay?

**Comment by kokomo123** [comment]
I really can't do that either way, I want to but github is refusing to reopen the PR.

**Comment by kokomo123** [comment]
Testing.

**Comment by kokomo123** [comment]
image

**Comment by shibaobun** [comment]
Ah, thank you for trying then

**Comment by shibaobun** [comment]
@kokomo123 Actually sorry, I'd really like to fix this issue. Is this PR able to be reopened now?

**Comment by kokomo123** [comment]
Thanks @nodiscc :)

**Comment by shibaobun** [comment]
Sorry about that, seems like it didn't work :pensive:
This PR should be ready to merge now though! If anyone is curious, they can email me at shibao@bubbletea.dev for more details

**Comment by shibaobun** [comment]
I think I've fixed the issues by the previous linter run. I just assumed that the tags were user-created, but now I've referenced it with Inventory Management, which makes more sense. I swear my PRs are usually not this messy :sweat_smile:

**Review by kokomo123 — APPROVED** [review]
LGTM, no issues here, pending further thoughts from collaborators. Thank you for your contribution!

File: software/cannery.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: "Cannery"
+website_url: "https://cannery.app"
+source_code_url: "https://gitea.bubbletea.dev/shibao/cannery"
+description: "A self-hosted firearm and ammunition tracker app."
```
Comment (Rabenherz112) [inline software/cannery.yml:4]: Small change to remove redundancy
```suggestion
description: "Firearm and ammunition tracker app."
```

**Review by Rabenherz112 — CHANGES_REQUESTED** [review]

**Review by Rabenherz112 — APPROVED** [review]

**Review by nodiscc — APPROVED** [review]

**Pull request merged**